src/hooks/useArticleLinksNavigation.js

```diff
@@ -101,6 +101,9 @@ export const useArticleLinksNavigation = (
 const makeLinkClickEvent = link => {
   const title = link.getAttribute('title')
   if (title) {
+    if (title.includes(':') && /^http(s?):\/\/(\w+)\.(\w+)\.org/.test(link.getAttribute('href'))) {
```

Comment:
I don't think this is enough. If a local link has a full url we'll open it in the browser instead of in the app, which would be pretty bad.